apps/website/src/pages/docs/variables.mdx

```diff
@@ -17,6 +17,12 @@ iTwinUI offers an extensive set of handcrafted CSS variables. If you're using iT
 
 We strongly encourage the use iTwinUI's CSS variables rather than hardcoding values. Many of the color values change depending on the theme or contrast settings. Because of this, we purposefully do not expose the variables' values on this site. However, if you need a particular variable's value, you can extrapolate it from [the raw CSS file](https://cdn.jsdelivr.net/npm/@itwin/itwinui-variables/index.css).
 
+## Border radius
```

Comment:
what's the benefit of alphabetically sorting? to me, it makes more sense to sort in a logical order / by importance.

Reply:
We can take a guess as the order of importance, but alphabetical at least makes scanning the TOC easier.  We have the left side nav sorted alphabetically as well.

Reply:
I agree with discussing it further internally.